src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -264,9 +304,7 @@ export function transform(formConfig, form) {
     delete clonedData.providerFacility;
   }
 
-  return JSON.stringify({
-    form526: JSON.stringify(clonedData),
-  });
+  return JSON.stringify({ form526: clonedData });
```

Comment:
We talked about this in Slack, but just for posterity, are we confident that `clonedData` shouldn't be stringified?

@SMLuthi, can you confirm that `form526` should be a regular ol' object and not a JSON string that needs to be parsed?

Reply:
Well, works locally at least. 🤷‍♂️